frontend/src/components/IntegrationConfiguration.tsx

```diff
@@ -218,49 +225,58 @@ const SelectBoard: FC<{ configuration: IntegrationConfiguration }> = ({
         <LoadingSpinner />
       ) : (
         <>
-          {(tokenError || errorMessage) && (
+          {(selectedError || boardsTokenError || errorMessage) && (
             <Alert
               severity={errorMessage ? 'error' : 'info'}
               onClose={() => {
-                if (tokenError) setTokenError(false);
+                if (boardsTokenError) setBoardsTokenError(false);
+                if (selectedError) {
+                  selected.refetch();
+                  setSelectedError(false);
+                }
                 if (errorMessage) setErrorMessage('');
               }}
               icon={false}
             >
               {errorMessage || t('Oauth can not select board')}
             </Alert>
           )}
-          <Select
-            name="board"
-            id="board"
-            className="react-select"
-            classNamePrefix="react-select"
-            styles={{ menuPortal: (base) => ({ ...base, zIndex: 9999 }) }}
-            placeholder="Select board"
-            isDisabled={boards.isError}
-            menuPortalTarget={document.body}
-            onChange={(entry) => {
-              if (entry) {
-                patch({ id, name, roadmapId, boardId: entry.value })
-                  .unwrap()
-                  .catch((err) => {
-                    setErrorMessage(
-                      err.data?.message ?? 'something went wrong',
-                    );
-                  });
-              }
-            }}
-            defaultValue={
-              selected.data && {
-                value: selected.data.id,
-                label: selected.data.name,
+          {selected.isSuccess && (
```

Comment:
```suggestion
          {!selected.isError && (
```

Sorry, this was my mistake in the previous review. I then realised that `selected.isSuccess` is too strict.
I guess `!selected.isError` may not be the most specific precondition for displaying this, but it should be reasonably good.